src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+import dto.CarStatus;
+
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(final List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<CarStatus> race() {
+        return cars.stream()
+                .peek(Car::move)
+                .map(Car::getStatus)
+                .toList();
+
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public int getMaxDistance() {
+        return cars.stream()
+                .mapToInt(car -> car.getStatus().distance())
```

Comment:
```suggestion
                .mapToInt(car -> car.getDistance())
```

[의견] 어떤 표현을 할 때 되도록 객체 구조를 보이지 않게 하는 것이 가독성을 개선할 수 있을 것 같아요.

Reply:
`객체 구조를 보이지 않게` 다시한번 기억하겠습니다!